docs/UserGuide.md

```diff
@@ -440,136 +574,46 @@ Optional identifiers can be added to show the list of matched clients with only
-* `INDEX` must be more than 1, and less than or equal to the index of the last item in the displayed list.
-
-**Examples**:
-* `delete 2` deletes the 2nd person in the currently displayed list.
-
-* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
-
-[Return to Table of Contents](#table-of-contents)
-<br><br>
-
-
-### `batch`: Execute commands in batch
-
-**Purpose**: Allows you to execute some commands in bulk, so that you do not need to repeatedly type in the same 
-commands.
-
-**Format**: `batch COMMAND INDICES [ARGUMENTS]`
-
-* Only `edit` and `delete` commands can be executed in batch.
-    * For more information on how these commands work and their parameters, refer to the
-      [`edit`](#edit-edit-client-contact) and [`delete`](#delete-delete-client-contact) sections accordingly.
-* `INDICES` are comma-separated e.g. `1, 2, 3`, and they refer to the index number shown in the displayed client list.
-* `INDICES` must all be more than 1, and less than or equal to the index of the last item in the displayed list.
-* The optional `ARGUMENTS` input is only applicable if the `COMMAND` is `edit`.
-* For `edit`, you can only batch edit the following attributes:
-    * phone number
-    * address
-    * tags
-    * insurance policies
-
-<div markdown="block" class="alert alert-info">
-:bulb: 
-**Tip**:
-The same rules apply when editing clients' phone number and address! <br>
-If a parameter is expected only once in the command, but you specified it multiple times, **only the last occurrence** of the parameter will be taken. 
-</div>
-
-**Examples**:
-* To batch edit the tags of more than 1 client contact.
-    * `batch edit 1, 2, 4 p/91234567 a/Hougang Green t/TanFamily i/FamPol#111`
-<br><br>
-* To batch delete more than 1 client contact.
-    * `batch delete 1, 2, 4`
-<br><br>
-      [Return to Table of Contents](#table-of-contents)
-<br><br>
-
-
 ### `sort`: Sort list of clients
 
-**Purpose**: Sorts the current list of clients in ClientBook.
+**Purpose**: Sorts the current list of clients in ClientBook so that you can view your clients in a particular order.
 
 **Format**: `sort -IDENTIFIER -DIRECTION`
 
 * Sorts the list of clients according to the specified `IDENTIFIER` and `DIRECTION`.
-* The specified `IDENTIFIER` can be `-n` to sort by name alphabetically or `-i` to sort by number of insurance policies, but not both.
+* The specified `IDENTIFIER` can be `-n` to sort your clients by name alphabetically or `-i` to sort by number of insurance policies, but not both.
 * The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order, but not both.
 
 **Examples**:
-* Sort the current list of clients by name in **descending** number of insurance policies.
+* If you want to see which of your clients have the most policies with you, sort the current list of clients 
+  in **descending** number of insurance policies.
```

Comment:
The spacing between this bullet and the next is less than the spacing between 2 consecutive bullets in the previous commands.

Reply:
Most of the commands have 1 line of spacing between consecutive examples, except for `find` command because it has many examples. Is there a specific command that you are referring to?